Package.swift

```diff
@@ -11,15 +11,34 @@ let package = Package(
     ],
     targets: [
         .target(name: "COpenCombineHelpers"),
-        .target(name: "OpenCombine", dependencies: ["COpenCombineHelpers"]),
+        .target(
+          name: "OpenCombine", 
+          dependencies: [
+            .target(name: "COpenCombineHelpers", condition: .when(platforms: [.macOS, .linux]))
```

Comment:
Let's not forget non-desktop systems here and below.

```suggestion
            .target(name: "COpenCombineHelpers", condition: .when(platforms: [.macOS, .iOS, .watchOS, .tvOS, .linux]))
```